### PR TITLE
Refactor/miaw mxplan

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

File: packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.controller.js [hunk 1]
```diff
@@ -22,6 +22,12 @@ export default class {
       this.block = true;
     }
 
+    if (!isUndefined(this.$attrs.convertToUcents) && this.$attrs.convertToUcents === '') {
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.controller.js]: I don't get this ?

If it's defined and the value is empty ? I feel like booleans would make things easier to write

`<my-component convert-to-ucent="{{ $ctrl.shouldConvertToUcents }}"></my-component>`

For consistency with HTML5 you would also have `convert-to-ucent="convert-to-ucent"` but who wants that, really

Comment (marie-j) [reply packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.controller.js]: This is to act like `disabled` or `required`, you don't have to pass any value if you want this attribute always active

Comment (FredericEspiau) [reply packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.controller.js]: And if I want to disable it dynamically I should do this

`<my-component convert-to-ucent="{{ $ctrl.shouldConvertToUcents && 'thisIsDisabled' }}"></my-component>`

😢

Comment (marie-j) [reply packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.controller.js]: Ok I understand what you mean now 😅

Comment (marie-j) [reply packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.controller.js]: @FredericEspiau I've now updated so both syntax can be used, you can have 
```suggestion
    <my-component convert-to-ucent></my-component>
```
or 
```suggestion
    <my-component convert-to-ucent="{{ $ctrl.shouldConvertToUcents }}"></my-component>
```

Comment (FredericEspiau) [reply packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.controller.js]: Genius

**Pull request merged**